recipes/thinc/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "thinc" %}
+{% set version = "5.0.8" %}
+{% set sha256sum = "993a12ab10c0af52c1db814fdabeecc591163e76c1fff50857ca9785c7a1841c" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256sum }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  skip: True  # [win32 or linux32]
+  features:
+    - vc9   # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py>34]
+
+# Numpy supported is >=1.7
+requirements:
+  build:
+    - toolchain
+    - msinttypes  # [win and py<35]
+    - python
+    - setuptools
+    - numpy x.x
+    - murmurhash >=0.26,<0.27
+    - cymem >=1.30,<1.32
+    - preshed >=0.46,<0.47
+
+  run:
+    - python
+    - numpy x.x
+    - murmurhash >=0.26,<0.27
+    - cymem >=1.30,<1.32
```

Comment:
> @ocefpaf I see. I don't mind watching for packages being built. Is that a hard rule?

Not a hard rule, that is why I merged it.

> There is a related PR #2455 that also includes a bunch of dependencies. I estimate it takes a minimum 3-5 days to get a recipe merged and the feedstock created.

Maybe b/c it is a bunch of recipes :smile: I rarely review many recipes. This batch you were in luck b/c I am waiting in line with nothing to do :wink: 

> So doing a separate PR for each small dependency would imply a longer wait time until we get the final package published.

I understand, but that is recommended.